spec/controllers/moab_storage_controller_spec.rb

```diff
@@ -56,6 +56,12 @@
         get :show, params: { id: fixture_druid }
         expect(assigns(:output)[:current_version]).to eq 10
       end
+
+      it 'object_size' do
+        get :show, params: { id: fixture_druid }
+        expect(assigns(:output)[:object_size]).to be_between(21_909_525, 22_049_525)
```

Comment:
consider making these round numbers:  21_909_000, 22_049_000   ???

Reply:
Yeah sounds good, I just did a ±70,000 bytes from the directory file size given by the method. But i'll change that.